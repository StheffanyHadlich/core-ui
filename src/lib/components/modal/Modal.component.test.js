import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/modal/__snapshots__"
});
