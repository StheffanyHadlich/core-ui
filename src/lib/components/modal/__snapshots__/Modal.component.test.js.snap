// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Modal Default 1`] = `
<div
  className="sc-fjdhpX fYqDPr"
>
  <div
    className="sc-modal sc-bdVaJa gIywMY"
  >
    <div
      className="sc-modal-content sc-bwzfXH fKuiVN"
    >
      <div
        className="sc-modal-header sc-htpNat fPOfap"
      >
        <span
          className="sc-bxivhb jmnWmC"
        >
          Hello
        </span>
        <div
          className="sc-bZQynM fBBzW"
          onClick={[Function]}
        >
          <i
            className="fas fa-times"
          />
        </div>
      </div>
      <div
        className="sc-modal-body sc-ifAKCX ftDzci"
      >
        <span>
          Do you want a cookie?
        </span>
      </div>
      <div
        className="sc-modal-footer sc-EHOje gzTeXM"
      >
        <div
          style={
            Object {
              "display": "flex",
              "justifyContent": "space-between",
            }
          }
        >
          <button
            className="sc-button sc-iwsKbI dzjUlM"
            disabled={false}
            onClick={[Function]}
            size="small"
            title=""
            type="button"
          >
            <span
              className="sc-VigVT gRqLsd"
            >
              <span
                className="sc-button-text"
              >
                <span
                  className="sc-gqjmRU icJnoE"
                >
                  No
                </span>
              </span>
            </span>
          </button>
          <button
            className="sc-button sc-iwsKbI bVCWGk"
            disabled={false}
            onClick={[Function]}
            size="small"
            title=""
            type="button"
          >
            <span
              className="sc-VigVT gRqLsd"
            >
              <span
                className="sc-button-text"
              >
                <span
                  className="sc-gqjmRU icJnoE"
                >
                  Yes
                </span>
              </span>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
